### ISSUE TITLE
WARNING: An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by jprokos** [filebot/filebot#49]
Is anyone else getting this warning?

```
$ filebot -script fn:sysinfo
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/usr/local/Caskroom/filebot/4.7.9/FileBot.app/Contents/Java/FileBot_4.7.9.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
FileBot 4.7.9 (r4984)
JNA Native: 5.1.0
MediaInfo: 0.7.93
7-Zip-JBinding: 9.20
Chromaprint: 1.4.2
Extended Attributes: OK
Unicode Filesystem: OK
Script Bundle: 2018-03-16 (r516)
Groovy: 2.4.10
JRE: Java(TM) SE Runtime Environment 9.0.1
JVM: 64-bit Java HotSpot(TM) 64-Bit Server VM
CPU/MEM: 4 Core / 6 GB Max Memory / 44 MB Used Memory
OS: Mac OS X (x86_64)
Package: APP
uname: Darwin iMac132.local 17.5.0 Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018; root:xnu-4570.51.1~1/RELEASE_X86_64 x86_64

-------------------- UPDATE AVAILABLE: FileBot 4.8 (r5280) ---------------------
```

I received a similar warning when testing the Plex script for Vuze as well.
``` `filebot -script fn:amc --output "/Volume/Movies/Seeding" --action copy -non-strict "/path/to/input" --log-file amc.log --def excludeList=amc.txt` ```

`Illegal Argument: java.nio.file.NoSuchFileException: /path/to/input (/path/to/input)
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/usr/local/Caskroom/filebot/4.7.9/FileBot.app/Contents/Java/FileBot_4.7.9.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Argument[0]: File does not exist: /path/to/input
Illegal usage: output folder must exist and must be a directory: /Volume/Movies/Seeding
-bash: Locking: command not found
`

**Comment by rednoah** [comment]
`FileBot 4.7.9` is not forwards-compatible with `Java 9`. Please use `Java 8`.

**Comment by rednoah** [comment]
`Illegal Argument: java.nio.file.NoSuchFileException: /path/to/input` is a completely unrelated problem. You're asking FileBot to process the file/folder `/path/to/input` and so of course it'll fail, and the error message couldn't be more clear:

```
Argument[0]: File does not exist: /path/to/input Illegal usage: output folder must exist and must be a directory
```

**Comment by jprokos** [comment]
I just copied your commands to test the AMC Plex Script and entered a valid path. You wrote this is what we should enter to test the AMC Plex script. I also have installed Java8 via Brew, but apparently there is a later version bundled in the App Store version of FileBot? Or maybe FileBot is not finding the Brew installed version of Java8?

Regardless, the error regarding reflective access is the main question and you chose not to address it.

**Comment by rednoah** [comment]
> FileBot 4.7.9 is not forwards-compatible with Java 9. Please use Java 8.

`Illegal reflective access` is caused by running `FileBot 4.7.9` with `Java 9`. The solution is to run `FileBot 4.7.9` with `Java 8`.

The MAS version of FileBot is self-contained and completely separate from any other applications and tools as per MAS rules. Hence, it is also updated separately (and more regularly) then the CLI tools managed by brew cask.

**Comment by jprokos** [comment]
The solution was actually to run these commands in terminal:
`java -version`
This told me what version was running as the default on my mac. I discovered it wasn't version 8 which I had downloaded via Brew, but version 9. This had me confused. Some searching revealed a command to see if there are more than one version installed:
`/usr/libexec/java_home -V`
Then some more searching revealed a method to set a system default Java version, by entering this command in .bash_profile or .bashrc in my User folder.
`export JAVA_HOME=`/usr/libexec/java_home -v 1.8``

Haven't tried this yet, just finished uninstalling Java completely.

Is there a way to have filebot choose version 1.8 while the system uses the latest, more secure version of Java 10? Is there a file we can modify within filebot package?
